input/pagecontent/terminology.md

```diff
@@ -8,72 +8,68 @@ This implementation guide does not define any unique value sets. The value sets
+|[ConditionClinicalStatusCodes](https://hl7.org/fhir/R4/valueset-condition-clinical.html)|[AU Core Condition](StructureDefinition-au-core-condition.html),|FHIR|
+|[ConditionVerificationStatus](https://hl7.org/fhir/R4/valueset-condition-ver-status.html)|[AU Core Condition](StructureDefinition-au-core-condition.html),|FHIR|
+|[DataAbsentReason](https://hl7.org/fhir/R4/valueset-data-absent-reason.html)|[AU Core Pathology Result Observation](StructureDefinition-au-core-diagnosticresult-path.html), [AU Core Heart Rate](StructureDefinition-au-core-heartrate.html), [AU Core BodyHeight](StructureDefinition-au-core-bodyheight.html), [AU Core Blood Pressure](StructureDefinition-au-core-bloodpressure.html), [AU Core Respiration Rate](StructureDefinition-au-core-resprate.html), [AU Core Body Weight](StructureDefinition-au-core-bodyweight.html), [AU Core Waist Circumference](StructureDefinition-au-core-waistcircum.html), [AU Core BodyTemperature](StructureDefinition-au-core-bodytemp.html), [AU Core Diagnostic Result Observation](StructureDefinition-au-core-diagnosticresult.html), [AU Core Smoking Status](StructureDefinition-au-core-smokingstatus.html)|FHIR|
+|[DVA Entitlement](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-dva-entitlement.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|AU Base|
+|[Encounter Reason Codes](https://hl7.org/fhir/R4/valueset-encounter-reason.html)|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|FHIR|
+|[EncounterStatus](https://hl7.org/fhir/R4/valueset-encounter-status.html)|[AU Core Encounter](StructureDefinition-au-core-encounter.html)|FHIR|
+|[EventStatus](https://hl7.org/fhir/R4/valueset-event-status.html)|[AU Core Procedure](StructureDefinition-au-core-procedure.html)|FHIR|
+|[Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1)|[AU Core Patient](StructureDefinition-au-core-patient.html)|NCTS|
+|[Healthcare Organisation Role Type](https://healthterminologies.gov.au/fhir/ValueSet/healthcare-organisation-role-type-1)|[AU Core Organization](StructureDefinition-au-core-organization.html)|NCTS|
+|[hl7VS-degreeLicenseCertificate - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base/ValueSet-au-v2-0360-extended.html)|[AU Core Practitioner](StructureDefinition-au-core-practitioner.html)|AU Base|
+|[hl7VS-identifierType - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v2-0203-extended.html)|[AU Core Organization](StructureDefinition-au-core-organization.html)|AU Base|
+|[hl7VS-providerRole - AU Extended](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-au-v2-0443-extended.html)|[AU Core Immunization](StructureDefinition-au-core-immunization.html)|AU Base|
+|[https://hl7.org/fhir/R4/valueset-identifier-use.html](https://hl7.org/fhir/R4/valueset-identifier-use.html)|[AU Core Patient](StructureDefinition-au-core-patient.html)|FHIR|
```

Comment:
URL (https://hl7.org/fhir/R4/valueset-identifier-use.html) shown as ValueSet name - change to 'IdentifierUse'

Reply:
Not sure what you meant by duplicate. The code system was incorrectly hyperlinked to the value set so I've swapped that out.

Reply:
I see two entries for IdentifierUse - the first has the ValueSet name as the URL (line 46), and the second entry has the name 'IdentifierUse' (line 47).

Reply:
removed he entry that had the value set url as the text to be displayed.